### DISCUSSION TITLE
Canadian PCB/Case Group Buy

### DISCUSSION THREAD

**Discussion by raychmond** [FundamentalFrequency/.github#13] [General]
Anyone who lives in Canada and would like to group buy, share ordering info or laser cutting experience can post here.

I live in Waterloo region and I am placing the order through JLCPCB right now. I may need 2 or 3 boards myself, so I can share the extra 2 - 3 PCBs with anyone in this region or even GTA.

I'm looking for the laser cutter in this region. I know there is 1 laser cutter in one of the local libraries, but I couldn't find it. 

Thanks,

**Comment by raychmond** [comment]
@stonepreston 
I found the 3-pcs transparent acrylic sheets on amazon.ca, link is [here](https://www.amazon.ca/Transparent-Acrylic-Plexiglass-Protective-Replacement/dp/B09XR6Y12Q/ref=sr_1_5?crid=23NJ06FTKEDUD&keywords=Acrylic%2BSheet%2B12%22%2Bx%2B16%22%2Bx%2B1%2F8%E2%80%9D&qid=1655757252&sprefix=acrylic%2Bsheet%2B12%2Bx%2B16%2Bx%2B1%2F8%2B%2Caps%2C133&sr=8-5&th=1)

Can you please confirm whether 3-pcs is good enough for the case? I know your BOM shows 4-pcs.

Thank you!

**Reply by stonepreston** [reply]
I used 4 pieces. 3 might be enough if you don't have to cut an extra layer ( I explain why you might need an extra layer in the MCAD repo). I think its highly likely you are going to need 4

**Reply by stonepreston** [reply]
that being said, you might have an acrylic distributor around you. Theres tons around me but I live in a large city. Thats cheaper than amazon for suer

**Reply by raychmond** [reply]
I will check that and share with our Canadian friends here.

**Comment by Dp666me** [comment]
I ordered a board but I havent gotten the case yet. I ordered mine from JLCPCB. For the case I hope to use a library that has one by me. I did contact a welding shop that had a acrylic cutter but they havent gotten a price for me yet. The hardest part has been the keys, and the screen I had to get them from here. https://shop.pimoroni.com/ it was the cheapest I could find even though it's from the UK

**Reply by Dp666me** [reply]
They came back and I have to change my order. Ill grab you a some boards and then I can mail it to you

**Reply by raychmond** [reply]
If you are talking about the key switches, I've got some 5-pin from amazon.ca ([link](https://www.amazon.ca/gp/product/B08JP84WZL/ref=ppx_yo_dt_b_asin_title_o07_s00?ie=UTF8&th=1)) and some 3-pin from aliexpress.com ([link](https://www.aliexpress.com/item/1005002581805632.html?spm=a2g0o.order_list.0.0.7681180227QkMr)). I think both should be fine. But I haven't confirmed yet.

You should definitely order the screen from pimoroni.com directly. They are the best with lowest shipping fee to Canada.

**Reply by stonepreston** [reply]
@raychmond Id go 5 pin. They provide much more stability and make alignment a lot easier, but 3 pins work too

**Reply by raychmond** [reply]
@stonepreston Will do 5 pins. Thank you for confirming that. I have plenty of key switches, diodes, encoders and joysticks. I happen to have a spare teensy 4.1 and a PI 4B. What a coincidence!

**Comment by kc3lai** [comment]
@raychmond I'm also in the Waterloo region.  Have you already placed order for the PCB? how much would it be for a board? I may get 2 or 3?

**Reply by raychmond** [reply]
@kc3lai I already placed the order. The total cost for 5 boards should be less than C$30 after $13 new user coupon. I will let you know the final price I paid. It’s great to hear we live in the same area. It’s much easier to group purchase.

**Reply by Dp666me** [reply]
They added an extra drilling charge the next day for me

**Reply by raychmond** [reply]
My order is in production progress now. They didn't add extra drilling charge, but there is a customs duties and taxes. The total now including shipping is C$34.42 after C$12.99 coupon.

**Reply by kc3lai** [reply]
@raychmond quick follow up on your order. has it arrived? :)

**Reply by raychmond** [reply]
Not yet. I chose the economic standard shipping, which takes about 3-4 weeks. But it costs way less than DHL. I will for sure let you know once I receive it.
Cheers,

**Reply by kc3lai** [reply]
awesome. thanks!

**Reply by raychmond** [reply]
@kc3lai Have you found how to make the case BTW?

**Reply by kc3lai** [reply]
nope not yet.  I can't start building until I get rpi4

**Reply by raychmond** [reply]
You can follow @rpilocator on Twitter. You may be able to buy one pretty quickly.

**Reply by raychmond** [reply]
@kc3lai 
adafruit.com may have rpi4B 4G today. They usually have stocks on Wednesday at 11:00am.

**Comment by Dylan-Grove** [comment]
I still have 4 PCBs left from my order if anyone wants one. Based in Alberta but I can ship if you pay for it.

**Reply by Dp666me** [reply]
> I still have 4 PCBs left from my order if anyone wants one. Based in Alberta but I can ship if you pay for it.

I'm in Alberta too. Why did you do for your case?

**Reply by Dylan-Grove** [reply]
> > I still have 4 PCBs left from my order if anyone wants one. Based in Alberta but I can ship if you pay for it.
> 
> I'm in Alberta too. Why did you do for your case?

I just printed it with my 3d printer. It's not super pretty and I had ro cut the prices in half in order to fit on the printer but I think it'll do. Can always upgrade to acrylic later.

If you want a PCB, I'm in Edmonton if you want to meet up!

**Reply by Dp666me** [reply]
> > > I still have 4 PCBs left from my order if anyone wants one. Based in Alberta but I can ship if you pay for it.
> > 
> > 
> > I'm in Alberta too. Why did you do for your case?
> 
> I just printed it with my 3d printer. It's not super pretty and I had ro cut the prices in half in order to fit on the printer but I think it'll do. Can always upgrade to acrylic later.
> 
> If you want a PCB, I'm in Edmonton if you want to meet up!

Yeah I have 5 in Edmonton already. Too bad we could of shared earlier. Could I buy a case off of you?

**Reply by raychmond** [reply]
Hopefully some people in Canada later could find our post here, so you guys can share your PCBs.

**Comment by raychmond** [comment]
I've got my 5 pc PCB yesterday. 
@kc3lai, how many do you want? Please leave a contact info, so we can discuss how you can pick it up?
@Dylan-Git @Dp666me I believe you guys all received your orders as well.

**Reply by kc3lai** [reply]
@raychmond I can be reached via kevin.c.lai at gmail.

**Comment by raychmond** [comment]
I still don't find readily available laser cutter locally. I'm 3D printing the cases right now. If it turns out to be successful, I will share the .stl files because I modified them to be able to 3D print on my printer (220mmx220mm). I think most people have this sized 3D printer.

**Reply by Dylan-Grove** [reply]
I 3d printed mine, just don't make the mistake I did of making the cut on the same spot on all of the parts. Makes it a little flimsy.

**Reply by Dp666me** [reply]
> I 3d printed mine, just don't make the mistake I did of making the cut on the same spot on all of the parts. Makes it a little flimsy.

What about having the top and bottom from a cut acrylic and printing the middle?

**Reply by Dylan-Grove** [reply]
> What about having the top and bottom from a cut acrylic and printing the middle?

That would work. Honestly you might want to redesign the STL files so that it's one solid peice. Having a bunch of layers doesn't look very good. 

I'm trying to design my own metal enclosure that would end up being thinner by like 10mm. It's going to require modifying the Pi to remove the USB ports and using a different touch screen, but I think it will look much better than the one in the build guide. Only problem is learning FreeCAD which is as easy as I had hoped.

**Comment by TVDMsfhs1** [comment]
@kc3lai, @Dylan-git, @Dp666me, @raychmond  - Sorry for the 4 way ping- I'm looking for a LMN3 PCB if anyone still has one. I'm in Calgary but I may be able to swing an Edmonton location if needed. timothyofthebuckets AT gmail dot com

**Reply by Dp666me** [reply]
Yeah I have one you can have for free in Edmonton

**Reply by Dylan-Grove** [reply]
Same. Also have lots of extra parts in Edmonton collecting dust

**Reply by TVDMsfhs1** [reply]
@Dylan-Grove @Dp666me that's Great . Any chance either of you are close to groat road and 118th NW? I have a cousin living there for a mail box drop off OR I could come grab it next time I'm up there.
i'd be happy to pay for the board + shipping too if your willing to throw it in the mail to me. What ever is easiest.

Email me if any of that works for you!

**Reply by Dp666me** [reply]
I can ship the board to you next weekend. I can't see your email?

**Reply by TVDMsfhs1** [reply]
> I can ship the board to you next weekend. I can't see your email?

Amazing - Thank you!
email: timothyofthebuckets AT gmail DOT com
If you email me, I will send you the address

**Reply by TVDMsfhs1** [reply]
@Dp666me just checking you got my email address from the last post. Excited to build this thing when it arrives. Thanks again

timothyofthebuckets AT gmail

**Reply by Dp666me** [reply]
I sent a few emails nothing back
